met-web/src/components/feedback/FeedbackModal/index.tsx

```diff
@@ -232,7 +232,10 @@ export const FeedbackModal = () => {
                                 <PrimaryButton
                                     data-testid="submit-button"
                                     loading={isSaving}
-                                    disabled={Boolean(!rating || (comment_type !== CommentTypeEnum.None && !comment))}
+                                    disabled={Boolean(
+                                        (rating === RatingTypeEnum.None && comment_type === CommentTypeEnum.None) ||
```

Comment:
can we extract these two into variables with meaningful names then use the variables here instead.

Reply:
updated to have the conditions in a variable. thanks.